src/Extractor/ContentExtractor.php

```diff
@@ -531,6 +531,15 @@ function ($element, $currentEntity) {
         // If no year has been found during date_parse, we nuke the whole value
         // because the date is invalid
         if (false === $parseDate['year']) {
+            $this->logger->info('Date is bad (wrong year): {date}', ['date' => $this->date]);
+
+            $this->date = null;
+        }
+
+        // in case the date can't be converted we assume it's a wrong date
+        if (0 > strtotime($this->date)) {
```

Comment:
`strtotime` returns `false` since PHP 5.1 :)

Reply:
But `strtotime('-0001-11-30T00:00:00+00:00')` return a negative value: https://3v4l.org/P40gB
But yeah, I should check for `false` too